src/Abstractions/NexusMods.Abstractions.NexusModsLibrary/Models/Collection.cs

```diff
@@ -0,0 +1,54 @@
+using NexusMods.Abstractions.MnemonicDB.Attributes;
+using NexusMods.Abstractions.NexusModsLibrary.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Models;
+
+namespace NexusMods.Abstractions.NexusModsLibrary.Models;
+
+/// <summary>
+/// Metadata about a collection on Nexus Mods.
+/// </summary>
+public partial class Collection : IModelDefinition
```

Comment:
For consistency, this should be named like the other models in the project.

Reply:
I disagree, I really don't like the other models in the project, and want to rename them. As it stands every model in there is named `NexusMods.Abstractions.NexusModsLibrary.NexusModsFileMetadata` or something like that, it's the Batman naming problem. I'd much more prefer to use C# namespacing to separate these models.

Reply:
The `NexusMods` prefix can go, but the `Metadata` suffix is still useful since this is remote metadata we pulled from Nexus Mods.

That way you have the collection loadout group and the collection remote metadata instead of the collection loadout group and the "collection".

Reply:
Sounds good, added `Metadata` to the model names